index.html

```diff
@@ -491,64 +485,80 @@ <h2>
-      navigables=], and it manages the state transitions between different
-      [=credential request coordinator/interaction states=].
-    </p>
-    <p>
-      Each [=top-level traversable=] has a single associated [=credential
-      request coordinator=].
+      is a user-agent-defined component that mediates [=digital credential=]
+      interactions within a [=top-level traversable=]. Each [=top-level
+      traversable=] has exactly one associated coordinator. The coordinator
+      ensures that at most one interaction is active across all [=child
+      navigables=], orchestrates the end-to-end presentation or issuance flow,
+      and manages transitions between [=credential request
+      coordinator/interaction states=]. A user agent may delegate some or all
+      coordinator responsibilities to external wallet applications, platform
+      components, or other trusted entities according to user or platform
+      policy.
     </p>
     <p>
-      The coordinator manages the lifecycle of the current interaction's
-      {{Promise}}, including its resolution or rejection and integrates with an
-      {{AbortSignal}} to allow for both user-initiated and programmatic
-      cancellation.
+      The coordinator manages the lifecycle of the interaction's {{Promise}}
+      and its associated {{AbortSignal}}, including resolution with the user's
+      selected [=digital credential=] (or credentials) or rejection due to
+      errors or user/programmatic cancellation.
     </p>
     <p>
-      A [=credential request coordinator=] has an associated [=credential
-      request client=], which it interacts with.
+      The [=credential request coordinator=]:
     </p>
+    <ul>
+      <li>Validates and transforms presentation or issuance inputs and outputs.
+      </li>
+      <li>Interprets each individual request in the input list as targeting a
+      specific [=holder=], and prepares [=holder=] options accordingly.
+      </li>
+      <li>Presents the available [=holders=] to the user for selection.
+      </li>
+      <li>Resolves the interaction's {{Promise}} with the selected [=digital
+      credential=] or rejects it to indicate cancellation.
+      </li>
+    </ul>
+    <aside class="note">
+      <p>
+        Although the coordinator handles input/output coordination, the
+        user-selected [=holder=] typically acts as the [=credential chooser=].
+      </p>
+    </aside>
     <h3>
       Interaction states
     </h3>
     <p>
       The [=credential request coordinator=] has a finite set of
       <dfn data-dfn-for="credential request coordinator">interaction
-      states</dfn>, which are used to manage the lifecycle of a request for a
-      digital credential, whether it is a [=digital credential/presentation
-      request=] or an [=digital credential/issuance request=]. The states are:
+      states</dfn> used to manage the lifecycle of a [=digital
```

Comment:
These two words should not be deleted.
```suggestion
      states</dfn>, which are used to manage the lifecycle of a [=digital
```